cmd/mesh/manifest-generate_test.go

```diff
@@ -25,11 +27,17 @@ import (
 type testGroup []struct {
 	desc       string
 	flags      string
+	noInput    bool
+	outputDir  string
 	diffSelect string
 	diffIgnore string
 }
 
 func TestManifestGenerateFlags(t *testing.T) {
+	outDirectory, err := ioutil.TempDir("", "output-testing")
```

Comment:
clean up the temp dir after test?

Reply:
I am cleaning up, but in `runTestGroup()` - which is not the right place. Will put it in here